tests/e2e/cli/instance_segmentation/test_tiling_instseg.py

```diff
@@ -43,7 +43,7 @@
     "train_params": [
         "params",
         "--learning_parameters.num_iters",
-        "10",
+        "1",
```

Comment:
Specific reason to fix?

Reply:
It's my mistake. Thanks for pointing it. I fixed it.